lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -628,7 +629,7 @@ defmodule Membrane.ICE.Endpoint do
 
       [
         demand: pad,
-        event: {pad, %Handshake.Event{handshake_data: hsk_data}}
+        event: {pad, to_srtp_keying_material_event(hsk_data)}
```

Comment:
- `hsk_data` can be `nil` and the conversion function doesn't support that
- could we keep the event in the state instead of generating it in all these places?